src/XrdCrypto/XrdCryptosslX509.cc

```diff
@@ -507,8 +507,8 @@ const char *XrdCryptosslX509::IssuerHash(int alg)
       if (issueroldhash.length() <= 0) {
          // Make sure we have a certificate
          if (cert) {
-            char chash[15] = {0};
-            snprintf(chash,15,"%08lx.0",X509_NAME_hash_old(cert->cert_info->issuer));
+            char chash[30] = {0};
+            snprintf(chash, sizeof(chash), "%08lx.0", X509_NAME_hash_old(cert->cert_info->issuer));
```

Comment:
I don't fully understand this string format. Why do you need _width_ set to 8 and also ".0" at the end. Just looks a bit awkward. Does your hash really end with ".0"?